Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/remesh.h

```diff
@@ -230,10 +231,12 @@ void isotropic_remeshing(const FaceRange& faces
 #ifdef CGAL_PMP_REMESHING_VERBOSE
     std::cout << " * Iteration " << (i + 1) << " *" << std::endl;
 #endif
-
-    remesher.split_long_edges(high);
-    remesher.collapse_short_edges(low, high);
-    remesher.equalize_valences();
+    if (target_edge_length!=0)
```

Comment:
what do you think of changing that with `if(target_edge_length > 0)` ?
Or maybe leave this, and add an assertion `CGAL_assertion(target_edge_length >= 0)` instead